README.md

```diff
@@ -4,7 +4,9 @@
 
 _NOTE: This is in beta and is expected to GA in Fall 2020._
 
-This is the Lightstep package for configuring OpenTelemetry
+This is the Lightstep package for configuring OpenTelemetry.
+
+There are two ways you can use this package - either as an agent JAR, or directly integrated in your Java code. You should select only one of these integration options, not both. If you'd like to extend the automatic instrumentation offered by the agent, then please see [this page](https://github.com/open-telemetry/opentelemetry-java-instrumentation#manually-instrumenting) for information on extending traces into your code.
```

Comment:
if they want to do both, they can use the launcher integration instructions here right (with launcher as the mvn dependency)? or do they still need the otel API as in this link?

Reply:
I believe they'd need to follow the instructions in the link and get the API (and, potentially, the extensions lib import for @WithSpan). Carlos would have a more authoritative answer I think?

Reply:
it's possible to use agent together with manual instrumentation, trick is to use the same tracer (https://github.com/lightstep/otel-launcher-java/tree/master/examples/agent)

Reply:
hm, looks like my previous comment about same tracer is not correct. but we need to test that when 0.8.0 will be released, currently 0.7.2 is not working due to incompatible api of agent and otel-java 07.0

Reply:
0.8.0 works fine